lib/utils/resolve-to-testing-library-fn.ts

```diff
@@ -171,11 +171,12 @@ export const resolveToTestingLibraryFn = <
 	}
 
 	const customModuleSetting = context.settings['testing-library/utils-module'];
-	if (
-		[...LIBRARY_MODULES, USER_EVENT_MODULE, customModuleSetting].some(
-			(module) => module === maybeImport.source
-		)
-	) {
+	const hasImportModuleMatch =
+		[...LIBRARY_MODULES, USER_EVENT_MODULE].includes(maybeImport.source) ||
+		(typeof customModuleSetting === 'string' &&
+			maybeImport.source.endsWith(customModuleSetting));
```

Comment:
There are several utils related to detect Testing Library modules in `lib/create-testing-library-rule/detect-testing-library-utils.ts`, perhaps we can reuse something here?

If not, we probably need to combine that module with this one at some point.